java_console/models/src/com/rusefi/core/Sensor.java

```diff
@@ -115,32 +121,22 @@
     @Nullable
     private final FieldType type;
     private final int offset;
+    private final double scale;
 
-    Sensor(String name, SensorCategory category, FieldType type, int offset, BackgroundColor color, double minValue, double maxValue, String units) {
+    Sensor(String name, SensorCategory category, FieldType type, int offset, double scale, BackgroundColor color, double minValue, double maxValue, String units) {
         this.name = name == null ? name() : name;
         this.type = type;
         this.offset = offset;
+        this.scale = scale;
         this.category = category;
         this.color = color;
         this.units = units;
         this.minValue = minValue;
         this.maxValue = maxValue;
     }
 
-    Sensor(SensorCategory category, FieldType type, int offset, BackgroundColor color, double minValue, double maxValue, String units) {
-        this(null, category, type, offset, color, minValue, maxValue, units);
-    }
-
     Sensor(String name, SensorCategory category, FieldType type, int offset, BackgroundColor color, double minValue, double maxValue) {
-        this(name, category, type, offset, color, minValue, maxValue, "n/a");
```

Comment:
some of these constructors were no longer necessary, as I added name/min/max/units to the channels that were missing it.